zap/src/main/java/org/parosproxy/paros/core/scanner/Analyser.java

```diff
@@ -141,7 +141,7 @@ private void addAnalysedHost(URI uri, HttpMessage msg, int errorIndicator) {
             mapVisited.put(uri.toString(), new SampleResponse(msg, errorIndicator));
 
         } catch (HttpMalformedHeaderException | DatabaseException e) {
-            logger.error("Failed to persist the message: " + e.getMessage(), e);
+            logger.error("Failed to persist the message: {}", e.getMessage(), e);
```

Comment:
These are being done in the other PR, should not conflict though.

Reply:
Ya, I’ll sort it out whichever lands first.